Source/WebGPU/WebGPU/Adapter.mm

```diff
@@ -130,9 +130,7 @@
 
 void Adapter::requestInvalidDevice(CompletionHandler<void(Ref<Device>&&)>&& callback)
 {
-    instance().scheduleWork([strongThis = Ref { *this }, callback = WTFMove(callback)]() mutable {
-        callback(Device::createInvalid(strongThis));
-    });
+    callback(Device::createInvalid(*this));
```

Comment:
I think we've already discussed this ... Device creation is supposed to be async.

Reply:
This code is not even used actually so I will remove it.